.travis.yml

```diff
@@ -5,30 +5,53 @@ language: generic
 env:
   - DOCKER_COMPOSE_VERSION=1.14.0
 
-install:
-  # Installing docker
+before_install:
+  # Add Docker APT repository
   - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
-  - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+  - sudo add-apt-repository "deb https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
   - sudo apt-get update
+
+install:
+  # Install Docker
   - sudo apt-get -y install docker-ce
 
-  # Installing docker-compose
-  - sudo rm /usr/local/bin/docker-compose
-  - curl -L "https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-$(uname -s)-$(uname -m)" > docker-compose
+  # Install Docker Compose
+  - curl -L "https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-$(uname -s)-$(uname -m)" -o docker-compose
   - chmod +x docker-compose
   - sudo mv docker-compose /usr/local/bin
 
-before_script:
-  # Check docker & docker-compose versions
+  # Print runtime versions
   - docker --version
   - docker-compose --version
 
-script:
+before_script:
+  # Build images
   - docker-compose build
-  - docker-compose up -d
 
-  # Verifications
-  - sleep 30
+  # Pre-pull buildpack image (curl 7.52.0+)
+  - docker pull buildpack-deps:artful-curl
```

Comment:
What is the purpose of this image?

Reply:
It contains a version of `curl` that supports `--retry-connrefused`, which is unfortunately not available on Ubuntu Trusty (base OS of Travis workers).